.github/workflows/push-build.yml

```diff
@@ -0,0 +1,8 @@
+name: Build Webapp on Push
+
+on:
+  push:
+
+jobs:
+  call-workflow-build:
+    uses: OpenOlitor/openolitor-client-kundenportal/.github/workflows/grunt-build.yml@fix/deps_node16
```

Comment:
16?

Reply:
This is the link to the file. Since it's only available in the current branch, I had to add this. As soon as on prod, I will change this and create another PR. 

And it's 16 because I had the plan to go to 16 but since 18 worked out, it's better.